rust/pact_verifier/src/callback_executors.rs

```diff
@@ -85,20 +85,22 @@ impl ProviderStateExecutor for HttpRequestProviderStateExecutor {
       Some(state_change_url) => {
         let mut state_change_request = Request { method: "POST".to_string(), .. Request::default() };
         if self.state_change_body {
-          let mut json_body = json!({
+          let json_body = json!({
                     "state".to_string() : json!(provider_state.name.clone()),
+                    "params".to_string() : json!(provider_state.params.clone()),
```

Comment:
I've discovered that the serde `json!` macro is recursive, so we don't need to repeat it on the values.